app/javascript/components/layout/Header.js

```diff
@@ -52,7 +53,12 @@ class Header extends React.Component {
               <Form inline className="ml-auto">
                 <Navbar.Text className="text-white py-0 px-3">
                   <i className="fas fa-user fa-fw mr-2"></i>
-                  {this.props.current_user?.email}
+                  {this.props.current_user?.email} (
+                  {this.props.current_user?.role
```

Comment:
Just add a quick check here to make sure the role is valid before performing operations on it. 
```
this.props.current_user?.role?.split....`
```

Reply:
Now added!